2-mandatory/2-financial-times.js

```diff
@@ -14,7 +20,13 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
```

Comment:
I like this implementation! Looks very good.
Just keep an eye on how you're naming variables. Developers use conventions for naming variables, functions, etc. that make it easier for other developers to read your code. I found some conventions listed on this page that you could take a look at: https://www.30secondsofcode.org/articles/s/javascript-naming-conventions